torchtitan/config_manager.py

```diff
@@ -207,6 +207,62 @@ def __init__(self):
             default=1,
             help="Pipeline Parallelism degree. 1 means disabled.",
         )
+        self.parser.add_argument(
+            "--experimental.pipeline_parallel_stages_per_rank",
+            type=int,
+            default=1,
+            help="""
+                Pipeline Parallelism number of stages per rank (a.k.a. virtual stages)
+
+                For simple schedules, this should be 1.
+
+                For looped schedules, this can be greater than one.
+
+                If the number of stages produced by splitting does not match the expected number of stages,
+                an error will be raised for sanity.""",
+        )
+        self.parser.add_argument(
+            "--experimental.pipeline_parallel_split_points",
+            type=int,
+            nargs="+",
+            default=[],
+            help="""
+                Specify names of modules to use as the beginning of a split point.
+
+                e.g. ["layers.1"] will cause the model to be split into 2 stages,
+                the first containing all the layers up to layers.0,
+                the second containing layers.1 and all the remaining layers.
+
+                Note: fully-automated splitting may be enabled in the future,
+                but currently the split points must be specified manually for both manual and tracer.""",
```

Comment:
oh, just saw this. If that's the case, either we can remove the `pipeline_parallel_stages_per_rank` option and print some thing to user, or keep it as sanity check. Maybe better to keep.